### ISSUE TITLE
no-unused-vars false error message

### ISSUE THREAD

**Issue by JustFly1984** [benmosher/eslint-plugin-import#140]
no-unused-vars false error message on second line "  messageItems,"

but it is defined on line  9 and later in code
```
let childElements = 0,
    messageItems,
    messageBoxHeight = 0,
    ids = [];
const removeMessage = (id) => {
  const message = fbi(id);
  if (message) removeElement(message);
  childElements = messageBox.childElementCount;
  messageItems = fbc("mmw");
  if (L && L.getItem(id)) L.removeItem(id);
  ids = ids.filter((e) => e != id);
  if (L && L.getItem("msg")) L.setItem("msg",ids);
  toggleMessagesButtonHandler();
  if (childElements === 0) hide(mms,mmb,camm);
  setMessageBoxHeight();
};
.........
```

**Comment by JustFly1984** [comment]
sorry, my bad
